server/helpers/ffmpeg-utils.ts

```diff
@@ -136,3 +134,18 @@ function getVideoFileStream (path: string) {
     })
   })
 }
+
+/**
+ * Standard profile, with variable bitrate audio and faststart.
+ *
+ * As for the audio, quality '5' is the highest and ensures 96-112kbps/channel
+ * See https://trac.ffmpeg.org/wiki/Encode/AAC#fdk_vbr
+ */
+function standard (ffmpeg) {
+  ffmpeg
+    .videoCodec('libx264')
+    .audioCodec('libfdk_aac')
```

Comment:
Maybe we could detect in the checker function that ffmpeg supports libfkd, and if not fallback to default aac encoder?

Reply:
If that's possible, that would be the best option. Any idea on how to do that?

*side note*: when we default back to the default aac encoder, we should also come back to CBR, as its support for VBR is infamous.

Reply:
I think you can use this function https://github.com/Chocobozzz/PeerTube/blob/develop/server/initializers/checker.ts#L50

See https://github.com/fluent-ffmpeg/node-fluent-ffmpeg#querying-ffmpeg-capabilities

Then store this information somewhere :)

Reply:
Any preference as to where to store the information? Because I think I shouldn't modify the CONFIG object just for that.

Reply:
You can create a function in ffmpeg-utils that create an object with information we want, and cache it for future calls?